editions/tw5.com/tiddlers/concepts/SystemTiddlers.tid

```diff
@@ -4,7 +4,7 @@ tags: Concepts
 title: SystemTiddlers
 type: text/vnd.tiddlywiki
 
-Tiddlers are classified as system tiddlers if their title starts with the special string `$:/`. The only difference from ordinary tiddlers is that they are hidden from most searches and lists in the sidebar. (In particular, system tiddlers are shown in the "More/System" list).
+Tiddlers are classified as system tiddlers if their title starts with the special string `$:/`. System tiddlers are hidden from most searches and lists in the sidebar. (In particular, system tiddlers are shown in the "More/System" list). A [[Module|Modules]] must be a system tiddler.
```

Comment:
It isn't correct that a module must be a system tiddler, there are no restrictions on their names, they are identified by the module-type field and the type being application/javascript

Reply:
This seemed to be true in my installation when I tested it, getting a simple "hello world" module to run. It could be that way because I'm running with `root-tiddler=$:/core/save/lazy-all`.